### PR TITLE
Overall

### PR DESCRIPTION
**Pull request by haylisha**
- [ ] PR has short and clear description
- [x] PR has appropriate labels and milestones for easy identification
- [x] PR it is assigned to the owner
- [x] reviewers are assigned
- [ ] the PR contributes only one focused change
- [ ] the branch is up to date with `main`/`master`
- [ ] the code works when pulled and run locally
- [ ] all conflicts are resolved (if any)
- [x] It is linked to an issue in the appropriate column of the project board
- [x] feedback is addressed (if any, and if it is appropriate feedback.)

## README Documentation

- [x] The markdown source is formatted
- [ ] Spelling and grammar is correct in all text
- [x] The markdown looks correct when you preview the file
- [ ] All links and images work
- [ ] The README documents the solution's behavior, strategy(ies) and implementation(s)
- [ ] There are use cases in code blocks to illustrate the function's behavior

## Python Files

- [ ] There is a module header
- [ ] There is a module docstring
- [ ] File names are in snake_case
- [ ] Test files are named `test_<module_name>.py`

## Function Docstring

- [ ] Behavior description
- [ ] Parameter description
- [ ] Return value description
- [ ] Include any assumptions
- [ ] Include 3 or more (passing!) doctests
- [ ] Include 1-2 use cases (if necessary)

## Function Implementation

- [ ] The code is auto-formatted
- [ ] The code has no (reasonable) linting mistakes
- [ ] Variables are named with snake_case
- [ ] The function has a clear and helpful name
- [ ] The file's name matches the function name
- [ ] The code follows the strategy as simply as possible
- [ ] Variable names are clear and helpful
- [ ] Comments explain the strategy (if necessary)
- [ ] There are type annotations

### CODE REVIEW

**Comment by haylisha** [comment]
Try to check and give comments ASAP and continue we don't have much time

File: README.md [hunk 1]
```diff
@@ -1,21 +1,28 @@
...
 
 
 ## Problem statement
 <!-- your comment -->
 Please follow the link to read full [problem statement]().
+Even though healthcare has made great strides, child mortality is still an international issue caused by differences in access to care, socio-economic status, and varying healthcare structures. To combat this problem effectively requires precise predictive models that analyze past data, identify influential factors and predict future patterns of child mortality rates. This initiative seeks to create a reliable system for predicting trends through machine learning algorithms which can expose the intricate dynamics behind children's deaths accurately. By doing so we aim at developing targeted interventions thereby enhancing informed policy making and ultimately steering down global infant death statistics considerably as well contribute significantly towards achieving our goals globally concerning saving more young lives from unnecessary childhood morbidity or/and depletion thereof altogether due to preventable causes which rob them of their rightful place in society both now but also into-extended futures not just locally but indeed internationally too!
```
Comment (indigo-tj) [inline README.md:14]: This idea is not clear, especially the last sentence. It's very long. I suggest changing it a little as an example like this:

```suggestion
While healthcare advancements have been significant, child mortality remains a pressing global concern, largely stemming from disparities in access to care, socio-economic status, and healthcare infrastructure. Addressing this challenge requires precise predictive models that analyze historical data, identify key factors, and forecast future trends in child mortality rates.

Our initiative aims to develop a reliable predictive system using machine learning algorithms to uncover the complex dynamics behind children's deaths. By doing so, we seek to implement targeted interventions, inform policy decisions, and significantly reduce global infant mortality rates. Ultimately, our goal is to save more young lives by preventing unnecessary childhood morbidity and mortality, thus ensuring a brighter future for children worldwide.
```
Or just rewrite it a little.

**Review by indigo-tj — CHANGES_REQUESTED** [review]
Thank you @haylisha for your work!
I have a small comment on the README file.

File: scripts/child mortality.ipynb
Comment (indigo-tj) [inline scripts/child mortality.ipynb:1]: Always use snake_case for naming. Can you please change file name?

File: README.md (hunk 1 above)
Comment (haylisha) [reply README.md:14]: it is very important, let it be for now we will modify it later on as we finalize our work

File: scripts/child mortality.ipynb
Comment (haylisha) [reply scripts/child mortality.ipynb:1]: ok

**Pull request closed without merging**